src/searchbar.cpp

```diff
@@ -164,9 +164,17 @@ void SearchBarLineEdit::focusInEvent( QFocusEvent* event)
     if (event->reason() == Qt::ActiveWindowFocusReason ||
         event->reason() == Qt::MouseFocusReason ||
         event->reason() == Qt::ShortcutFocusReason) {
+        connect(&m_completer, QOverload<const QString &>::of(&QCompleter::activated),
+        this, &QLineEdit::setText,Qt::UniqueConnection);
+
         connect(&m_completer, QOverload<const QModelIndex &>::of(&QCompleter::activated),
         this, QOverload<const QModelIndex &>::of(&SearchBarLineEdit::openCompletion),
         Qt::UniqueConnection);
+
+        connect(&m_completer, QOverload<const QModelIndex &>::of(&QCompleter::highlighted),
+        this, [=](const QModelIndex &index){
+            setText(index.isValid() ? index.data().toString() : m_searchbarInput);
+        }, Qt::UniqueConnection);
```

Comment:
Indent the 2nd and subsequent lines of every `connect()` expression so that  they are aligned with the first argument on the first line (fix this formatting in the earliest commit where the respective expression is modified).

Reply:
Done